heartbeat/iSCSILogicalUnit.in

```diff
@@ -491,6 +500,14 @@ iSCSILogicalUnit_stop() {
 		ocf_release_lock_on_exit $TARGETLOCKFILE
 		# "targetcli delete" will fail if the LUN is already
 		# gone. Log a warning and still push ahead.
+		
+		local backstore_type
+		if targetcli /iscsi/${OCF_RESKEY_target_iqn}/tpg1/luns/lun${OCF_RESKEY_lun} status | grep ' fileio/' >/dev/null; then
```

Comment:
Wouldnt the `if [ -b` test above work here as well?

Reply:
The rationale behind this is that the path of the backing store may not exist in the filesystem at this point thus checking its type may fail, while this will still work.

Reply:
Ah. Makes sense.